ui/lib/core/addon/components/search-select.js

```diff
@@ -172,7 +177,26 @@ export default Component.extend({
     return filterOptions(options || [], searchText, matcher);
   },
   // -----
-
+  customizeObject(option) {
+    // if passObject=true return array of objects
```

Comment:
It's not an array, it's an object with the selected values, right? I'm not sure I fully understand the nuances here, and I think the comments might be a bit misleading. Pretty sure any of the models that this component fetches a list for results in `option` being a model record, it's just a matter of whether some attributes were hydrated.

Reply:
The comment here was leftover, I think, from when this logic was in the `onChange` method and it was just reiterating what `passObject` does as a param. Will remove as I think it's confusing now..